letsencrypt-auto-source/Dockerfile.centos6

```diff
@@ -0,0 +1,32 @@
+# For running tests, build a docker image with a passwordless sudo and a trust
+# store we can manipulate.
+
+FROM centos:6
+
+RUN rpm -ivh http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
```

Comment:
I just tested it and it works. The only caveat is you can't do one `yum install` step like:
```
yum install -y epel-release python-pip sudo
```
`python-pip` is in EPEL which needs to be completely installed before you can install `python-pip`.